gradle/maven-publish.gradle

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
The steps in this file is the same as for other pravega repositories. No changes have been made other naming.